app/src/main/java/org/oppia/app/walkthrough/WalkthroughPageChanger.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.walkthrough
+
+/** Listener for when an activity should change pages. */
```

Comment:
Maybe: ``... should change pages during the app walkthrough flow.`` This isn't so generic to represent all multi-page activities, just the walkthrough. We should make that clear in the documentation.

Ditto for other interfaces. Make sure to be specific about what sort of effect the interface should have on the app.

Reply:
Okay!